strings/blind.txt

```diff
@@ -227,3 +227,11 @@ very chic
 Zombie geese
 are waterfoul.
 -
+I bought some
+cuccos for a
+poultry sum.
+-
+The status of
```

Comment:
perhaps `The stratus of` ?

Reply:
I vote more layers as opposed to less. I think the "stratus" line was actually intended when VT pointed it out.

Reply:
I'd be willing to go with "stratus", then.

Reply:
I kinda like stratus too.